libs/common/src/lib/interfaces/user-settings.interface.ts

```diff
@@ -11,5 +11,6 @@ export interface UserSettings {
   language?: string;
   locale?: string;
   savingsRate?: number;
+  targetNetWorth?: number;
```

Comment:
Rename accordingly

Reply:
used projectedTotalAmount in the new commit